src/ui/Pricing/Comparison/index.svelte

```diff
@@ -10,6 +10,7 @@
   export { className as class }
   export let plans: SAN.Plan[]
   export let isShowingMore = false
+  export let isLoggedIn = false
```

Comment:
Add `isEligibleForTrial` prop which should be passed to `Plan` components 
```suggestion
  export let isLoggedIn = false
  export let isEligibleForTrial: boolean
```